robottelo/helpers.py

```diff
@@ -400,18 +406,11 @@ def prepare_import_data(tar_path=None):
         cmd = u'tar -xzvf {0} -C {1}'.format(tar_path, tmpdir)
 
     files = {}
-    for file in ssh.command(cmd).stdout:
-        for key in (
-            'activation-keys',
-            'channels',
-            'config-files-latest',
-            'kickstart-scripts',
-            'repositories',
-            'system-groups',
-            'system-profiles',
-            'users'
-        ):
-            if file.endswith(key + '.csv'):
-                files[key] = join(tmpdir, file)
+    for filename in ssh.command(cmd).stdout:
+        for key in ('activation-keys', 'channels', 'config-files-latest',
+                    'kickstart-scripts', 'repositories', 'system-groups',
+                    'system-profiles', 'users'):
+            if filename.endswith(key + '.csv'):
+                files[key] = join(tmpdir, filename)
```

Comment:
this refactor is done to address a pylint error and to fix the usage of `file` keyword